### PR TITLE
[trantor] Update to 1.5.14

### PR DESCRIPTION
**Pull request by an-tao**

### CODE REVIEW

File: ports/trantor/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "trantor",
-  "version-semver": "1.5.11",
+  "version-semver": "1.5.12",
   "port-version": 1,
```
Comment (JonLiu1993) [inline ports/trantor/vcpkg.json:4]:
```suggestion
```

File: ports/trantor/portfile.cmake [hunk 2]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO an-tao/trantor
-    REF v1.5.11
-    SHA512 dd65938bebb2e6714e5603db3bfc82cd1a63395c17dce014147a41fdc74548cb526e1457a7472aa51bb80ce629a9935b4db9eeadf735efaf30899ef73f776a58
+    REF v1.5.12
```
Comment (JonLiu1993) [inline ports/trantor/portfile.cmake:4]:
```suggestion
    REF "v${VERSION}"
```

**Comment by JonLiu1993** [comment]
Please run command "./vcpkg x-add-version trantor --overwrite-version" and commit again.

File: ports/trantor/vcpkg.json [hunk 3]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "trantor",
-  "version-semver": "1.5.11",
-  "port-version": 1,
+  "version": "1.5.14",
```
Comment (JonLiu1993) [inline ports/trantor/vcpkg.json:3]: @an-tao, could you please clarify why the new version change to "version"?

File: versions/t-/trantor.json [hunk 4]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "b90025f7c3ca717353d6e6f8b932156c1fdce3fa",
+      "version-semver": "1.5.14",
```
Comment (JonLiu1993) [inline versions/t-/trantor.json:5]: And should remain the same as the `vcpkg.json` file.

File: ports/trantor/vcpkg.json (hunk 3 above)
Comment (an-tao) [reply ports/trantor/vcpkg.json:3]: Sorry，I don’t quite understand the difference between them. I tried several times and only this change can pass the CI check.

Comment (JonLiu1993) [reply ports/trantor/vcpkg.json:3]: You can reference [this](https://learn.microsoft.com/en-us/vcpkg/users/versioning).

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
@an-tao, When I tested the usage by `trantor:x64-windows`:
````
trantor provides CMake targets:

  # this is heuristically generated, and may not be correct
  find_package(Trantor CONFIG REQUIRED)
  target_link_libraries(main PRIVATE Botan::Botan Trantor::Trantor)
````
I get this error:
````
1> [CMake] -- Detecting CXX compile features - done
1> [CMake] -- Found OpenSSL: optimized;F:/Feature-test/trantor/vcpkg/installed/x64-windows/lib/libcrypto.lib;debug;F:/Feature-test/trantor/vcpkg/installed/x64-windows/debug/lib/libcrypto.lib (found version "3.1.2")  
1> [CMake] -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
1> [CMake] -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
1> [CMake] -- Looking for pthread_create in pthreads
1> [CMake] -- Looking for pthread_create in pthreads - not found
1> [CMake] -- Looking for pthread_create in pthread
1> [CMake] -- Looking for pthread_create in pthread - not found
1> [CMake] -- Found Threads: TRUE  
1> [CMake] -- Configuring done (13.3s)
1> [CMake] CMake Error at usageTest/CMakeLists.txt:9 (target_link_libraries):
1> [CMake]   Target "test" links to:
1> [CMake] 
1> [CMake]     Botan::Botan
1> [CMake] 
1> [CMake]   but the target was not found.  Possible reasons include:
1> [CMake] 
1> [CMake]     * There is a typo in the target name.
1> [CMake]     * A find_package call is missing for an IMPORTED target.
1> [CMake]     * An ALIAS target is missing.
````

CMakeLists.txt:
````
cmake_minimum_required (VERSION 3.8)

project(test)

find_package(Trantor CONFIG REQUIRED)
# Add source to this project's executable.
add_executable (test "usageTest.cpp")

target_link_libraries(test PRIVATE Botan::Botan Trantor::Trantor)
````

**Comment by an-tao** [comment]
Please try the following:
```
find_package(Trantor CONFIG REQUIRED)
  target_link_libraries(main PRIVATE Trantor::Trantor)
```

**Comment by JonLiu1993** [comment]
> Please try the following:
> 
> ```
> find_package(Trantor CONFIG REQUIRED)
>   target_link_libraries(test PRIVATE Trantor::Trantor)
> ```

The test was successful. It seems that `Botan::Botan` should not be included in the provided usage.

**Comment by an-tao** [comment]
> > Please try the following:
> > ```
> > find_package(Trantor CONFIG REQUIRED)
> >   target_link_libraries(test PRIVATE Trantor::Trantor)
> > ```
> 
> The test was successful. It seems that `Botan::Botan` should not be included in the provided usage.

How was that prompt generated?

**Comment by JonLiu1993** [comment]
> Please try the following:
> 
> ```
> find_package(Trantor CONFIG REQUIRED)
>   target_link_libraries(test PRIVATE Trantor::Trantor)
> ```
We should consider why Trantor::Trantor is not found instead of removing it.

**Comment by dg0yt** [comment]
> We should consider why Trantor::Trantor is not found instead of removing it.

You should consider why it is in the usage output for trantor.

**Comment by BillyONeal** [comment]
I don't think we need to be testing usage on every single update when the edits are only like-in-kind, which is why I merged https://github.com/microsoft/vcpkg/pull/34119 which already had the effect of this update. (I didn't know this PR existed at the time, sorry!)

Thanks for your contribution!

**Pull request closed without merging**